bbc/simorgh
# Fix article ID regex matching

Overall changes
======
- Fix Optimo ID regex path matching check to be much more specific to `/articles/`
- Add optional chaining to `filterForBlockType` function to handle null arrays

Testing
======
1. Visit http://localhost:7080/hausa/news/2014/07/140704_chimpanzee_communication?renderer_env=live
2. Confirm it loads correctly
3. Visit http://localhost:7080/hausa/labarai-53617732?renderer_env=live
4. Confirm the page loads (video player may not as media appears to have been withdrawn)

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
